src/web3_extentions/tx_utils.py

```diff
@@ -47,7 +47,7 @@ def sign_and_send_transaction(
             {
                 "from": account.address,
                 "gas": gas_limit,
-                "maxFeePerGas": pending_block.baseFeePerGas * 2 + self.w3.eth.max_priority_fee,
```

Comment:
![image](https://user-images.githubusercontent.com/27773025/218053582-1d39a6a7-846b-4dfc-ae91-52b6dabfe1d9.png)

Reply:
yea, I also doubt about it
there's a typing warning because returned type is a TypedDict
but actually, it's AttributedDict

Reply:
reverted